utils_nlp/eval/rouge_ext.py

```diff
@@ -0,0 +1,978 @@
+        else:
+            self.stemmer = RougeExt.STEMMER_DICT[self.language]
+
+        # # Load static objects
+        # if len(Rouge.WORDNET_KEY_VALUE) == 0:
+        #     Rouge.load_wordnet_db(ensure_compatibility)
+        # if Rouge.STEMMER is None:
+        #     Rouge.load_stemmer(ensure_compatibility)
+
+    # @staticmethod
+    # def load_stemmer(ensure_compatibility):
+    #     """
+    #     Load the stemmer that is going to be used if stemming is enabled
+    #     Args
+    #         ensure_compatibility: Use same stemmer and special "hacks" to product
+    #             same results as in the official perl script (besides the number of
+    #             sampling if not high enough)
+    #     """
+    #     Rouge.STEMMER = nltk.stem.porter.PorterStemmer('ORIGINAL_ALGORITHM') if
+    #         ensure_compatibility else nltk.stem.porter.PorterStemmer()
+
+    # @staticmethod
+    # def load_wordnet_db(ensure_compatibility):
+    #     """
+    #     Load WordNet database to apply specific rules instead of stemming + load file for
+    #     special cases to ensure kind of compatibility (at list with DUC 2004) with the
+    #     original stemmer used in the Perl script
+    #     Args
+    #         ensure_compatibility: Use same stemmer and special "hacks" to product same
+    #             results as in the official perl script (besides the number of sampling
+    #             if not high enough)
+
+    #     Raises:
+    #         FileNotFoundError: If one of both databases is not found
+    #     """
+    #     files_to_load = [Rouge.WORDNET_DB_FILEPATH]
+    #     if ensure_compatibility:
+    #         files_to_load.append(Rouge.WORDNET_DB_FILEPATH_SPECIAL_CASE)
+
+    #     for wordnet_db in files_to_load:
+    #         filepath = pkg_resources.resource_filename(__name__, wordnet_db)
+    #         if not os.path.exists(filepath):
+    #             raise FileNotFoundError("The file '{}' does not exist".format(filepath))
+
+    #         with open(filepath, 'r', encoding='utf-8') as fp:
+    #             for line in fp:
+    #                 k, v = line.strip().split(Rouge.WORDNET_DB_DELIMITER)
+    #                 assert k not in Rouge.WORDNET_KEY_VALUE
+    #                 Rouge.WORDNET_KEY_VALUE[k] = v
+
+    def tokenize_text(self, text):
+        """
+        Tokenize text in the specific language
+
+        Args:
+          text: The string text to tokenize
+          language: Language of the text
+
+        Returns:
+          List of tokens of text
+        """
+        return self.word_tokenize(text, self.language)
+
+    def split_into_sentences(self, text):
+        """
+        Split text into sentences, using specified language.
+
+        Args:
+          text: The string text to tokenize
+          language: Language of the text
+
+        Returns:
+          List of tokens of text
+        """
+
+        return self.sentence_split(text, self.language)
```

Comment:
is single line function needed?

Reply:
Good point! This function was in the original code and I prefer keeping it to reduce changes to the original code.